h2o-py/h2o/frame.py

```diff
@@ -1093,11 +1093,6 @@ def __getitem__(self, item):
         fr._ex._cache.types = new_types
         return fr
 
-    # def __getattr__(self, key):
```

Comment:
There could be a middle-ground solution for example if we allow such access for column names starting with a capital letter, and make a rule never to add any methods that start with a capital letter -- but I'm not sure how useful this will be.